src/platform/forms-system/src/js/components/ErrorableCheckbox.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash/fp'; // eslint-disable-line no-restricted-imports
```

Comment:
adding lodash/fp back and ignoring the eslint rule

Reply:
We can probably switch this back to the formation-react version at some point.

Reply:
I'll make a ticket to do this.